src/Symfony/Component/Serializer/Normalizer/GetSetMethodNormalizer.php

```diff
@@ -47,7 +47,7 @@ public function denormalize($data, $class, $format = null, array $context = arra
         $normalizedData = $this->prepareForDenormalization($data);
 
         $reflectionClass = new \ReflectionClass($class);
-        $object = $this->instantiateObject($normalizedData, $class, $context, $reflectionClass, $allowedAttributes);
+        $object = $this->instantiateComplexObject($normalizedData, $class, $context, $reflectionClass, $allowedAttributes, $format);
```

Comment:
@dunglas as said, my worries is that it's still a BC. If you have:

```php
class MyCustomNormalizer extends AbstractObjectNormalizer
{
  protected instantiateObject(..., $allowedAttributes, array $myParam = [])
  {
  }
}
```

Then you will have `$format = $myParam` which obviously will cause problems.